gatsby-config.js

```diff
@@ -93,5 +92,16 @@ module.exports = {
                 policy: [{ userAgent: '*', allow: '/' }],
             },
         },
+        {
+            resolve: `gatsby-plugin-netlify`,
+            options: {
+                headers: {
+                    '/page-data/*': [
+                        'cache-control: max-age=0,no-cache,no-store,must-revalidate',
```

Comment:
In the preview of this PR the `cache-control` seems to be as production? if it is the same why do we need this?